nixos/modules/services/logging/pushlog.nix

```
@@ -0,0 +1,171 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.pushlog;
+  unitType = types.submodule {
+    options = {
+      match = mkOption {
+        type = types.str;
+        default = ".*";
+      };
+      priorities = mkOption {
+        type = types.listOf types.int;
+        default = [
+          0
+          1
+          2
+          3
+          4
+          5
+          6
+        ];
```

Can be simplified using `lib.lists.range`

```suggestion
        default = lib.lists.range 0 6;
```